tests/fixtures/no-internal/rush.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
nuke this file, we are not using rush at all in this repo

Reply:
It's fake for testing purposes to recreate a monorepo/workspace situation. The adjacent directories workspace-pkg-1, 2, and 3 are all used in tests, and having a rush.json allows the [workspace-tools](https://www.npmjs.com/package/workspace-tools) package used in the rule to parse package.json files easily. Let me know if you think it's a bad idea to include this file though even if it's just for testing